package.json

```diff
@@ -21,39 +21,55 @@
     "useTabs": true,
     "tabWidth": 4,
     "singleQuote": true,
-    "bracketSpacing": false
+    "bracketSpacing": false,
+    "trailingComma": "none",
+    "arrowParens": "avoid"
   },
   "dependencies": {
     "@babel/core": "^7.8.3",
     "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-decorators": "^7.8.3",
     "@babel/plugin-proposal-export-default-from": "^7.8.3",
     "@babel/plugin-proposal-export-namespace-from": "^7.8.3",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.8.3",
+    "@babel/plugin-proposal-numeric-separator": "^7.8.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.8.3",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.8.3",
     "@babel/plugin-transform-react-inline-elements": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
     "@enact/dev-utils": "^2.4.1",
     "@storybook/addon-actions": "^5.3.5",
     "@storybook/addon-info": "^5.3.5",
     "@storybook/addon-knobs": "^5.3.5",
     "babel-plugin-dev-expression": "^0.2.2",
     "less": "^3.10.3",
-    "less-loader": "^5.0.0",
+    "less-loader": "^6.0.0",
     "mini-css-extract-plugin": "^0.9.0",
     "postcss-flexbugs-fixes": "^4.1.0",
     "postcss-global-import": "^1.0.6",
     "postcss-loader": "^3.0.0",
+    "postcss-normalize": "^9.0.0",
     "postcss-preset-env": "^6.7.0",
     "postcss-resolution-independence": "^1.0.0",
     "react-dev-utils": "^10.0.0"
   },
   "devDependencies": {
     "@storybook/react": "^5.3.5",
-    "babel-eslint": "^10.0.3",
+    "babel-eslint": "^10.1.0",
     "babel-loader": "^8.0.6",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.9.0",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-prettier": "^3.1.2",
-    "prettier": "^1.19.1",
+    "eslint": "^7.2.0",
+    "eslint-config-enact": "enactjs/eslint-config-enact#feature/rule-update",
```

Comment:
I'm concerned about basing this on a branch rather than a release.

Reply:
Yea, a few PRs have `eslint-config-enact` release as a requirement and this field will be updated before merge